src/js/modules/core/components/PageFooter.js

```diff
@@ -128,55 +126,25 @@ const styles = {
   },
 };
 
-const PageFooter = ({
-  classes,
-  topLevelSectionsWithChildren,
-  descriptions,
-}) => {
-  const createSectionLinks = () => {
-    return Object.values(topLevelSectionsWithChildren).map(section => {
-      return (
-        <div className={classes.sectionBlock} key={section.id}>
-          <Link
-            className={classes.topLevelSectionLink}
-            key={section.id}
-            to={section.permalink}
-          >
-            {section.name}
-          </Link>
-          {Object.values(section.subsections).map(subsection => {
-            return (
-              <Link
-                className={classes.subsectionLink}
-                key={subsection.id}
-                to={subsection.permalink}
-              >
-                {subsection.name}
-              </Link>
-            );
-          })}
-        </div>
-      );
-    });
-  };
+const PageFooter = ({ classes, sections, descriptions }) => {
   const createDescriptionLinks = () => {
-    const descriptionLinks = Object.values(descriptions).map(description => {
-      return (
-        <Link
-          className={classes.subsectionLink}
-          key={description.id}
-          to={`/about/${description.slug}`}
-        >
-          {description.title}
-        </Link>
-      );
-    });
+    // is a function because it is reused
```

Comment:
Instead of using a function, instantiate `createDescriptionLinks()` in an array before component load and reuse the `array.map`.